charter.md

```diff
@@ -1,39 +1,54 @@
 # MASQUE Charter
 
 Many network topologies lead to situations where transport protocol proxying is beneficial. For example, proxying
-enables endpoints to communicate when end-to-end connectivity is not possible, or to apply additional encryption where
-desirable (such as a VPN). Proxying can also improve client privacy, e.g., by hiding a client's IP address from a target
-server.
+enables endpoints to communicate when end-to-end connectivity is not possible or to apply additional encryption where
+desirable (such as a VPN). Proxying can also improve client privacy, e.g., by hiding a client's IP address from a
+target server.
 
 Proxying technologies such as SOCKS and HTTP(S) CONNECT exist, albeit with their own shortcomings. For example, SOCKS
 signalling is not encrypted and HTTP CONNECT is currently limited to TCP. In contrast, HTTP/3 is a viable candidate
-protocol for proxying arbitrary traffic, as in a single connection it provides secure connectivity, multiplexed streams,
-address migration, and a unified congestion controller. An HTTP/3 datagram construct built on top of QUIC datagram
-frames would provide for unreliable data transmission and enable transporting UDP and other unreliable flows via a
-proxy. Moreover, it would not introduce potentially redundant or unnecessary recovery mechanisms. Lastly, HTTP supports
+protocol for proxying arbitrary traffic, as it provides a single connection with secure connectivity, multiplexed
+streams, address migration, and a unified congestion controller. An HTTP/3 datagram construct built on top of QUIC
+datagram frames provides for unreliable data transmission and enables transporting UDP and other unreliable flows via a
+proxy. Moreover, it avoids introducing potentially redundant or unnecessary recovery mechanisms. Lastly, HTTP supports
 an established request/response semantic that can set up and configure flows for different services.
 
```

Comment:
These sentences are redundant with the third paragarph.

Reply:
Updated in [e06ed3e](https://github.com/ietf-wg-masque/wg-materials/pull/7/commits/e06ed3e41b0802294715b3b6bf26c6938828132c)